vnext/ReactUWP/Views/TextInputViewManager.cpp

```diff
@@ -374,14 +374,14 @@ void TextInputShadowNode::updateProperties(const folly::dynamic &&props) {
   auto textBox = control.try_as<winrt::TextBox>();
   auto passwordBox = control.try_as<winrt::PasswordBox>();
 
+  TryUpdateFontProperties(control, props);
```

Comment:
>TryUpdateFontProperties [](start = 2, length = 23)

check return value?

Reply:
just saw that this function now returns void. So the name of the function is wrong, it shouldn't have Try in it if it doesn't return a bool :)

---
In reply to: [371631838](https://github.com/microsoft/react-native-windows/pull/3830#discussion_r371631838) [](ancestors = 371631838)